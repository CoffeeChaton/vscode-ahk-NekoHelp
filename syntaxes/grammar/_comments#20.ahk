#Requires AutoHotkey v1.1.33+
#Warn All, MsgBox

autoRun()

autoRun() {
    i := 0
}

neverRun() {
    a := 0
    if (a > 0) {
        sText := "; ;"
        ;          ^^^ this comment, not string =_=, thank you ahk.
        sText := "; `;"
    }
}