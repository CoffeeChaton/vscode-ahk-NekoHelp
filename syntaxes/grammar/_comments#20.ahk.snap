>#Requires AutoHotkey v1.1.33+
#^^^^^^^^^ source.ahk string.parameter.directives.ahk keyword.control.directives.ahk
#         ^^^^^^^^^^^^^^^^^^^^ source.ahk string.parameter.directives.ahk
>#Warn All, MsgBox
#^^^^^ source.ahk string.parameter.directives.ahk keyword.control.directives.ahk
#     ^^^^^^^^^^^^ source.ahk string.parameter.directives.ahk
>
>autoRun()
#^^^^^^^ source.ahk entity.name.function.ahk
#       ^ source.ahk punctuation.bracket.ahk
#        ^ source.ahk punctuation.bracket.ahk
>
>autoRun() {
#^^^^^^^ source.ahk entity.name.function.ahk
#       ^ source.ahk punctuation.bracket.ahk
#        ^ source.ahk punctuation.bracket.ahk
#         ^ source.ahk
#          ^ source.ahk punctuation.bracket.ahk
>    i := 0
#^^^^^^ source.ahk
#      ^^ source.ahk keyword.operator.assignment.ahk
#        ^ source.ahk
#         ^ source.ahk constant.numeric.decimal.ahk
>}
#^ source.ahk punctuation.bracket.ahk
>
>neverRun() {
#^^^^^^^^ source.ahk entity.name.function.ahk
#        ^ source.ahk punctuation.bracket.ahk
#         ^ source.ahk punctuation.bracket.ahk
#          ^ source.ahk
#           ^ source.ahk punctuation.bracket.ahk
>    a := 0
#^^^^^^ source.ahk
#      ^^ source.ahk keyword.operator.assignment.ahk
#        ^ source.ahk
#         ^ source.ahk constant.numeric.decimal.ahk
>    if (a > 0) {
#^^^^ source.ahk
#    ^^ source.ahk keyword.control.flow.ahk
#      ^ source.ahk
#       ^ source.ahk punctuation.bracket.ahk
#        ^^ source.ahk
#          ^ source.ahk keyword.operator.comparison.ahk
#           ^ source.ahk
#            ^ source.ahk constant.numeric.decimal.ahk
#             ^ source.ahk punctuation.bracket.ahk
#              ^ source.ahk
#               ^ source.ahk punctuation.bracket.ahk
>        sText := "; ;"
#^^^^^^^^^^^^^^ source.ahk
#              ^^ source.ahk keyword.operator.assignment.ahk
#                ^ source.ahk
#                 ^ source.ahk string.quoted.double.ahk punctuation.definition.string.begin.ahk
#                  ^ source.ahk string.quoted.double.ahk
#                   ^ source.ahk string.quoted.double.ahk
#                    ^^ source.ahk meta.comment.line.semicolon.issues20.ahk comment.line.semicolon.issues20.ahk
>        ;          ^^^ this comment, not string =_=, thank you ahk.
#^^^^^^^ source.ahk
#       ^ source.ahk meta.comment.line.semicolon.ahk
#        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk meta.comment.line.semicolon.ahk comment.line.semicolon.ahk
>        sText := "; `;"
#^^^^^^^^^^^^^^ source.ahk
#              ^^ source.ahk keyword.operator.assignment.ahk
#                ^ source.ahk
#                 ^ source.ahk string.quoted.double.ahk punctuation.definition.string.begin.ahk
#                  ^^ source.ahk string.quoted.double.ahk
#                    ^^ source.ahk string.quoted.double.ahk constant.character.escape.ahk
#                      ^ source.ahk string.quoted.double.ahk punctuation.definition.string.end.ahk
>    }
#^^^^ source.ahk
#    ^ source.ahk punctuation.bracket.ahk
>}
#^ source.ahk punctuation.bracket.ahk